src/cohort-manager.js

```diff
@@ -55,6 +55,14 @@ class CohortManager {
     if (!foundStudent) throw new Error('student not found')
     return foundStudent
   }
+
+  addStudentToCohort(studentId, cohort) {
```

Comment:
Hadn't thought of doing it that way, that's an interesting one! 

One alternative, which I've gone for, is to have this method be part of the Cohort() class. I wonder what the merits of both solutions might be, and whether there's a best practice here. 

Lemme know if you've got any thoughts on that one!

Reply:
I did this from the manager class because I wanted students to be first entered at the school level and not a specific cohort. This was we can add students to maybe different cohorts that can be created. Like 'Cohort - data analysis' or 'Cohort - front-end'. Now my remove function can map through all cohorts this student belongs to and remove them from all of them or even just one. 

I know this will have to change when I implement the extension requirements.....

Reply:
I get that - mine ended up having some duplicate searchBy... (id, first name, last name, etc) methods so that both the whole 'school' and individual cohorts could be searched, but not all cohorts at the same time*. As you've said, this only matters if the extension requirements are implemented, namely the whole thing about one student only being part of one cohort at a time.

(*Essentially, I wanted to create a parent class and extend it, so I could store those methods there and have them available to both classes. Unfortunately, one of the classes involved already extends another.. and that's when I was reminded that one class can only extend one other class, no more)

Reply:
This is a good approach @PeachyOmnivore and a good demonstration of what a Manager class is. It manages other classes, and yours is doing it well